### PR TITLE
Added docs for StreamDeck4J.java

### PR DESCRIPTION
**Pull request by taydev**

### CODE REVIEW

File: src/main/java/com/walshydev/streamdeck4j/StreamDeck4J.java [hunk 1]
```diff
@@ -399,8 +431,16 @@ public void setImage(@Nonnull String context, BufferedImage image, String type,
         setImage(context, Base64.getEncoder().encodeToString(imageBytes), type, destination);
     }
 
+    /**
+     * Sets the image of a specific button on the Stream Deck
```
Comment (WalshyDev) [inline src/main/java/com/walshydev/streamdeck4j/StreamDeck4J.java:435]: Probably worth mentioning that the base64 encoded string doesn't need to have the `data:image/{type};base64,` at the start as I will append it.

File: src/main/java/com/walshydev/streamdeck4j/StreamDeck4J.java [hunk 2]
```diff
@@ -411,28 +451,58 @@ public void setImage(@Nonnull String context, String base64Encoded, String type,
...
+     * @param context The unique identifier of the button to show the checkmark on
+     */
     public void showOk(@Nonnull String context) {
         logger.trace("showOk(context)");
         sendEvent(SDEvent.SHOW_OK, null, context);
     }
 
+    /**
+     * Saves persistent data for the instance of the action. 
+     * ELI5: saves some data to the stream deck so if you restart it, it's still there
```
Comment (WalshyDev) [inline src/main/java/com/walshydev/streamdeck4j/StreamDeck4J.java:479]: Say how to get this data, it's in an event (you'll need to find which one)

File: src/main/java/com/walshydev/streamdeck4j/StreamDeck4J.java [hunk 3]
```diff
@@ -444,6 +514,13 @@ public void sendToPropertyInspector(@Nonnull String context, @Nonnull String act
         sendPayload(eventJson);
     }
 
+    /**
+     * Changes to a profile on the Stream Deck based on the profile's name
+     * 
+     * @param context The unique ID for the plugin, set as the PluginUUID received during registration.
```
Comment (WalshyDev) [inline src/main/java/com/walshydev/streamdeck4j/StreamDeck4J.java]: I should just handle this part myself, feel free to implement this. The UUID is stored in this class

**Review by WalshyDev — CHANGES_REQUESTED** [review]

File: src/main/java/com/walshydev/streamdeck4j/StreamDeck4J.java (hunk 3 above)
Comment (taydev) [reply src/main/java/com/walshydev/streamdeck4j/StreamDeck4J.java]: Removed context arg from that method and replaced it with `pluginUUID.toString()`

**Comment by taydev** [comment]
Better? @WalshyDev

**Review by WalshyDev — APPROVED** [review]

**Pull request merged**